build.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+
+# Copyright (c) 2019, NVIDIA CORPORATION.
+
+# rmm build script
+
+# This script is used to build the component(s) in this repo from
+# source, and can be called with various options to customize the
+# build as needed (see the help output for details)
+
+# Abort script on first error
+set -e
+
+NUMARGS=$#
+ARGS=$*
+
+# Get root of git repository without assuming location of build.sh script
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
```

Comment:
It depends, it is good to have there for safety. It can fail without dev/null if user has done some fancy escaping/redirecting output to stderr and/or massaged CDPATH. Btw. Even above line is not foolproof, it can still fail if user has done some symlink stuff to directory in question